src/lightmap.cpp

```diff
@@ -303,7 +300,7 @@ void map::generate_lightmap( const int zlev )
     const int minz = zlevels ? -OVERMAP_DEPTH : zlev;
     const int maxz = zlevels ? OVERMAP_HEIGHT : zlev;
     // Iterate top to bottom because sunlight cache needs to construct in that order.
-    for( int z = maxz; z >= minz; z-- ) {
+    for( int z = maxz; z >= zlev; z-- ) {
```

Comment:
What is the purpose of this change? I'm pretty sure it means sunlight can't penetrate lower than the level the player is standing on.

Reply:
My bad, for some reason I thought generate_lightmap() was being called for each zlevel. I'll revert it.